### PR TITLE
Add user title on forum post info

### LINKED ISSUE #5612: Add user title on forum
Seems like it would work nicely, especially for people with some kind of custom title, would mix great with usergroup badges informing about usergroup so it's not confusing, some examples of how i see it in mind, would be probably pretty much straightforward to implement if deemed fitting:

 

It's probably also to note it would have a good informing role for support team

### PR DESCRIPTION
**Pull request by venix12**
Fixes #5612

### CODE REVIEW

File: resources/views/forum/topics/_post_info.blade.php [hunk 1]
```diff
@@ -40,6 +40,12 @@ class="forum-post-info__row forum-post-info__row--username js-usercard"
             data-user-id="{{$user->user_id}}"
             href="{{ route("users.show", $user) }}"
         >{{ $user->username }}</a>
+
+        @if ($user->title())
```
Comment (nanaya) [inline resources/views/forum/topics/_post_info.blade.php]: Although usually fine, conditional test using plain string in php should be avoided. Use `isset` check.

Comment (venix12) [reply resources/views/forum/topics/_post_info.blade.php]: seems like ``isset`` won't work in this context, will go for a regular check instead lol

**Comment by nanaya** [comment]
You need to either allow edits from maintainers or update your branch with master.

**Comment by venix12** [comment]
uh yes, seems like for some reason i didn't do it 🤔 both done anyways

**Comment by nanaya** [comment]
Except it's already outdated :thinking:

**Pull request merged**